jobs/topline_dashboard.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -x
+
+if [[ -z "$mode" ]]; then
+   echo "Missing arguments!" 1>&2
+   exit 1
+fi
+
+bucket="net-mozaws-prod-metrics-data"
```

Comment:
I think files uploaded to this bucket need to have some special headers applied in order to propagate to the dashboard as expected. See [this line](https://github.com/mozilla-services/data-pipeline/blob/master/reports/executive_summary/run.sh#L124) for example.

We may need to explicitly copy the CSV file to the right location as part of the job to ensure it works.